lib/linter-registry.js

```diff
@@ -71,8 +74,7 @@ export default class LinterRegistry {
       return false
     }
 
-    const scopes = editor.scopeDescriptorForBufferPosition(editor.getCursorBufferPosition()).getScopesArray()
-    scopes.push('*')
+    const scopes = editor.scopeDescriptorForBufferPosition(editor.getCursorBufferPosition()).getScopesArray().concat(['*'])
```

Comment:
What about using this to support all cursors instead of just the last one?

```js
const scopes = editor.getCursors().reduce((scopes, cursor) => (
    scopes.concat(cursor.getScopeDescriptor().getScopesArray())
  ), []).concat(['*']);
```

_Note: This list isn't unique, there will likely be duplicates if there is more than one active cursor._